package-lock.json

```diff
@@ -11164,13 +11164,6 @@
       "integrity": "sha512-QSvevZ+IRww2ldtfv1QskYsqVVVwCKQf1XbwtcyyoRvLIQzfyPhj/C+3+PKzSDRdiyejaiLgnq//XTkleorpLg==",
       "dev": true
     },
-    "node_modules/@types/resize-observer-browser": {
```

Comment:
This shouldn't affect resize-observer-browser. When I run npm install, it only updates lines 2 and 7 for me.